gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/GemFireCacheImpl.java

```diff
@@ -602,11 +603,24 @@ public void removeRegionFromOldEntryMap(String regionPath) {
     }
   }
 
+  public long getOldEntryRemovalPerid() {
+    return OLD_ENTRIES_CLEANER_TIME_INTERVAL;
+  }
```

Comment:
We have to find a balance between frequency of thread run and amount of temporary memory overhead.